extension/src/popup/components/accountHistory/HistoryItem/styles.scss

```diff
@@ -1,54 +1,117 @@
 .HistoryItem {
-  color: var(--sds-clr-gray-12);
   cursor: pointer;
 
+  &__row {
+    display: flex;
+    flex: 1;
+    flex-direction: row;
+  }
+
   &__loader {
     display: flex;
     flex-direction: column;
     align-items: center;
     justify-content: center;
   }
 
-  &__row {
-    display: flex;
+  &__description {
+    flex: 1;
+    color: var(--sds-clr-gray-12);
+  }
+
+  &__amount {
+    text-align: right;
+    max-width: 45%;
   }
 
   &__icon {
-    border: 1px solid var(--sds-clr-gray-03);
-    border-radius: 2rem;
+    position: relative;
     display: flex;
     align-items: center;
     justify-content: center;
     height: 2rem;
-    margin-right: 1rem;
     width: 2rem;
+    border-radius: 2rem;
+    margin-right: 1rem;
+
+    &__swap-source {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 1;
+      height: 1.5rem;
+      width: 1.5rem;
+      border-radius: 1.5rem;
```

Comment:
same comment as above about pxToRem. I'm thinking we can chip away at replacing all `rem` with `pxToRem` over time

Reply:
https://github.com/stellar/freighter/pull/1785/commits/5d27d1c61f356cee0c34c44a7f08b64d768ac6ab